rider-core/src/main/java/com/github/database/rider/core/dataset/DataSetExecutorImpl.java

```diff
@@ -799,13 +799,13 @@ public void executeScript(String scriptPath) {
     private String[] readScriptStatementsFromFile(URL resource) {
         File scriptFile = getFileFromURL(resource);
         if (scriptFile == null) return null;
-        RandomAccessFile rad = null;
+        BufferedReader br = null;
         int lineNum = 0;
         try {
-            rad = new RandomAccessFile(scriptFile, "r");
+            br = new BufferedReader(new InputStreamReader(new FileInputStream(scriptFile), StandardCharsets.UTF_8));
```

Comment:
Can we use try-with-resources here?

Then we can get ride of the finally block

Reply:
Yes, we can, if we don't support JRE version below or equals 1.6
It depends on the java compile version

Reply:
Yes, we are compatible with java 1.7 onwards, [see here](https://github.com/database-rider/database-rider/blob/4c37cc7857de89310f3656d456815216fe2fde9a/pom.xml#L140-L141). 

Can you make that change?

Thanks!

Reply:
My pleasure! 
I've changed the relative code using try-with-resources